service/src/main/java/uk/gov/hmcts/reform/fpl/NotifyTemplates.java

```diff
@@ -44,8 +44,14 @@ private NotifyTemplates() {
     public static final String AMENDED_APPLICATION_RETURNED_CAFCASS_TEMPLATE = "d2e44b62-436c-4bb6-b3b9-afa4f2bca7b5";
     public static final String NOTICE_OF_NEW_HEARING = "c5525d1b-6431-489d-be0e-51de3f163583";
     public static final String SDO_AND_NOP_ISSUED_CTSC = "5ddf3927-f102-4c81-900a-57d8facac1d0";
+    public static final String URGENT_AND_NOP_ISSUED_CTSC = "309c7e39-b1e9-4a30-9d50-fa00d8d87e1c";
+    public static final String SDO_ISSUED_CTSC = "ea694754-d05e-46e2-85e5-f36dd25008fd";
     public static final String SDO_AND_NOP_ISSUED_LA = "6d0316f4-f58f-4ae8-ae25-a344854ac494";
+    public static final String URGENT_AND_NOP_ISSUED_LA = "3314bb72-f4b2-4c59-b67e-83825c44a030";
+    public static final String SDO_ISSUED_LA = "66806269-561c-4ea4-bced-701f0c126e4d";
     public static final String SDO_AND_NOP_ISSUED_CAFCASS = "e6292279-cc27-42d3-92ca-f75da3e18681";
```

Comment:
personally these should just be one template with a condition to show the NoP content or not, as we basically duplicate 3 templates to include very minor wording changes - but apparently it was problematic with UX to come up with the right wording for a combined template

Reply:
yeah we tried but it was decided that it would be more work to try and make them one template for each user, and then we would want to replace the existing templates as well which would require more work